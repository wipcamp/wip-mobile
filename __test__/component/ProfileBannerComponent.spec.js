/**
 * @jest-environment node
 */

import React from 'react'
import renderer from 'react-test-renderer'

import ProfileBanner from '../../components/ProfileBannerComponent'

it('rendering', () => {
    const tree = renderer
        .create(
            <ProfileBanner />
        )
        .toJSON()
    expect(tree).toMatchSnapshot()
})