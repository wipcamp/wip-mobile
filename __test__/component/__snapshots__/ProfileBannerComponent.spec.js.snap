// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 0.3,
      },
      Object {
        "flexDirection": "column",
      },
      Object {
        "backgroundColor": "#FFF",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "height": 22.5,
          "width": 100,
        },
        Object {
          "overflow": "hidden",
        },
      ]
    }
  >
    <Image
      source={1}
      style={
        Object {
          "height": 22.5,
          "resizeMode": "cover",
          "width": 100,
        }
      }
    />
  </View>
  <View
    style={
      Array [
        Object {
          "height": 7.5,
          "width": 100,
        },
        Object {
          "backgroundColor": "#FFF",
        },
      ]
    }
  />
  <Image
    source={undefined}
    style={
      Object {
        "borderRadius": 7.5,
        "bottom": 0,
        "height": 15,
        "left": 42.5,
        "position": "absolute",
        "width": 15,
        "zIndex": 100,
      }
    }
  />
</View>
`;
